gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValue.java

```diff
@@ -392,8 +414,10 @@ public BigDecimal getBigDecimal() {
       decimal = Bytes.toBigDecimal(rawBytes, OFFSET, rawBytes.length - OFFSET);
     } else if (isFloat()) {
       decimal = BigDecimal.valueOf(Bytes.toFloat(rawBytes, OFFSET));
-    } else if (isDouble())  {
+    } else if (isDouble()) {
       decimal = BigDecimal.valueOf(Bytes.toDouble(rawBytes, OFFSET));
+    } else if (isShort()) {
+      decimal = BigDecimal.valueOf(Bytes.toShort(rawBytes, OFFSET));
```

Comment:
I think we can use `getShort()` instad of `Bytes.toShort(rawBytes, OFFSET)` because thats's exactly what `getShort()` does.